gwlfe/WriteOutputFiles.py

```diff
@@ -7,9 +7,717 @@
 
 
 def WriteOutput(z):
-    print('WriteOutput')
+    # DIMENSION VARIABLES FOR PREDICT CALCULATION AND SCENARIO FILE
+    AvOtherLuSed = 0
+    AvOtherLuNitr = 0
+    AvOtherLuPhos = 0
+    TotSewerSys = 0
+    TotNormSys = 0
+    TotShortSys = 0
+    TotSeptSys = 0
+    TotAvLuErosion = 0
+    AvTotalSed = 0
+    AvDisN = 0
+    AvTotalN = 0
+    AvDisP = 0
+    AvTotalP = 0
+    n2t = 0
+    n6t = 0
+    n13t = 0
+    n24t = 0
```

Comment:
Glad these are defined as local variables. I put everything I encountered into that global data model and it was probably way more than need be.